tests/bsim/bluetooth/audio/src/cap_commander_test.c

```diff
@@ -833,11 +841,19 @@ static void test_change_volume(void)
 	err = bt_cap_commander_change_volume(&param);
 	if (err != 0) {
 		FAIL("Failed to change volume: %d\n", err);
-		return;
+		return err;
 	}
+	return param.volume;
+}
+static void test_change_volume(void)
+{
+	int new_volume;
 
+	UNSET_FLAG(flag_volume_changed);
+	new_volume = init_change_volume();
 	WAIT_FOR_FLAG(flag_volume_changed);
-	printk("Volume changed to %u\n", param.volume);
+
+	printk("Volume changed to %u\n", new_volume);
```

Comment:
What is the purpose of adding the return value? On error `FAIL` is already called

Reply:
To keep the behaviour exactly the same as before, but I am happy to simplify.

Reply:
Not sure what you mean by this. The only thing the return value adds is for printing purposes; it doesn't affect behavior.

Reply:
I'll remove the printk